Build reusable Team and User component for TeamUsers and Teams screens.

htichcock opened the pull request.

ianhitchcock committed:
Build reusable Team and User component for TeamUsers and Teams screens.

mandiwise commented on js/screens/TeamUsers/TeamUsersContainer.js:
```diff
@@ -0,0 +1,23 @@
+    ...headerBarStyle(navigation)
+  });
+
+  constructor() {
```
Useless constructor. Please remove.

mandiwise commented on js/screens/TeamUsers/TeamUsers.js:
```diff
@@ -0,0 +1,9 @@
+import TeamUserCard from "../../components/TeamUserCard";
+
+const TeamUsers = () => (
+  <TeamUserCard onPress={() => {}} fun={3} grow={4} win={3} />
```
Why are you passing an empty function here?

ianhitchcock committed:
Fix Merge Conflict.

htichcock commented:
TeamUsers was just used as a testing ground. The point of the PR was to get TeamUserCard on develop for others to use. Removed constructor and fixed a merge conflict.
